returnn/datasets/basic.py

```diff
@@ -736,13 +721,6 @@ def has_ctc_targets(self):
     """
     return False
 
-  # noinspection PyMethodMayBeStatic
-  def get_max_ctc_length(self):
-    """
-    :rtype: int
-    """
-    return 0
-
```

Comment:
In the comments, you suggest to also clean up CTC related stuff, but it sounds as if this is not done yet. However, here you partially remove some of it (and likely already break old Theano CTC setups by that). Is this on purpose? I don't quite understand why to remove only some part of it and not all (e.g. why not `has_ctc_targets` above?), or otherwise keep it as it is?

Reply:
`get_max_ctc_length()` was only used here:
```
device.alloc_data(shapes=shapes, max_ctc_length=dataset.get_max_ctc_length())
```
(in `theano/engine_util.py`)
But it always returned 0, which is also the default value for the `max_ctc_length` parameter. So this should not break anything, except maybe for custom datasets which actually properly implement this function. I doubt that anyone uses one, but I can't be sure.